### DISCUSSION TITLE
Device sometimes hangs after power outage

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#11794] [Q&A] [answered]
Hello, 

One of my tasmota devices does not boot after power outage. I have to turn the power OFF then ON to boot it again. I was trying to figure out why it is happening. I think it may be because AMS1117-3.3 regulator or 5v, 700ma power supply is not able to handle the load. Only thing out of the ordinary I can see in the web console is: 

20:08:41.455 QPC: Reset

what does QPC:Reset mean?

Regards,

Navi

**Comment by barbudor** [comment, marked as answer]
QPC : Quick Power Cycle, the recovery procedure that is triggered by multiple power cycles in a short time.
QPC Reset means that the current restart has not been identified as part of a short cycle and as such the counter is reset.
Should occur on any "normal" power restart.

**Reply by NsinghP** [reply]
Thanks,

So does that mean that I need to have the Web Console on before it crashes? or I need to hook up a serial console to record what is going on.

**Reply by sfromis** [reply]
I'd not really expect you to get any further clues in the console. Sometimes electronics goes into some sort of zombie state if bad things happens on power. And `QPC: Reset` is indeed just normal.

**Reply by NsinghP** [reply]
Thanks, I will replace the AMS chip and Power supply. Hopefully, it is just a faulty AMS1117 chip.
